website/content/en/docs/operator-scope.md

```diff
@@ -100,77 +92,6 @@ With the above changes the specified manifests should look as follows:
       name: memcached-operator
       namespace: <operator-namespace>
     ```
-
```

Comment:
Just moved to be ref.

Reply:
Where is the reference to that new doc?
Shouldn't we be linking to it from here?